integration_tests/main.rs

```diff
@@ -0,0 +1,276 @@
+pub(crate) mod common;
+
+use rand::SeedableRng;
+use zeekoe::{
+    customer::{self, database::StateName as CustomerStatus, zkchannels::Command},
+    merchant::{self, zkchannels::Command as _},
+    protocol::ChannelStatus as MerchantStatus,
+};
+
+use {
+    anyhow::Context,
+    common::{customer_cli, merchant_cli, LogType, Party},
+    rand::prelude::StdRng,
+    std::{fs::OpenOptions, panic, time::Duration},
+    structopt::StructOpt,
+    thiserror::Error,
+};
+
+#[tokio::main]
+pub async fn main() {
+    let rng = StdRng::from_entropy();
+    let server_future = common::setup(&rng).await;
+    let customer_config = customer::Config::load(common::CUSTOMER_CONFIG)
+        .await
+        .expect("Failed to load customer config");
+    let merchant_config = merchant::Config::load(common::MERCHANT_CONFIG)
+        .await
+        .expect("Failed to load merchant config");
+
+    // Give the server some time to get set up
+    tokio::time::sleep(tokio::time::Duration::new(5, 0)).await;
```

Comment:
You  made  a comment about this as well I believe, is this something we can do based upon the logging? I believe the server prints a certain message when it's ready...

Reply:
Yes! It required more code than I expected, but I added this functionality.